commercetools-models/src/main/java/io/sphere/sdk/carts/Cart.java

```diff
@@ -149,6 +149,15 @@ public String toString() {
     @Override
     @HasUpdateAction(value = "applyDeltaToLineItemShippingDetailsTargets", fields = {@PropertySpec(name = "lineItemId",type = String.class),@PropertySpec(name = "targetsDelta",type = ItemShippingTarget[].class)})
     @HasUpdateAction(value = "setLineItemShippingDetails", fields = {@PropertySpec(name = "lineItemId",type = String.class),@PropertySpec(name = "shippingDetails",type = ItemShippingDetailsDraft.class)})
+    @HasUpdateAction(value = "removeLineItem", makeAbstract = true,
```

Comment:
Cool! I wasn't even aware that we already support generation of abstract update actions 👍

Reply:
To be honest i forgot i added support for it myself 😄